db/schema.rb

```diff
@@ -10,11 +10,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170212201320) do
+ActiveRecord::Schema.define(version: 20170213013948) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "experiments", force: :cascade do |t|
+    t.integer "user_id"
```

Comment:
See [this SO post](http://stackoverflow.com/a/32021851/1516373). `belongs_to` in the model will tell `ActiveRecord` to use a foreign key to resolve the other end of the relationship based on standard naming conventions, but the database itself must enforce foreign key integrity